### ISSUE TITLE
Crop sampling in training + Inference(validation)

### ISSUE THREAD

**Issue by ZhengGang85129** [ZhengGang85129/DashCam#17]
Apply the following preprocessing step for the training dataset, 
During training (only on training dataset, don't do this on validation dataset):
(1) Resizing the shorter side of the video to a random value in [256, 320]. That means, 1280 x 720 (originally) ~ > 455 × 256
(2) Randomly sample a 224 x 224 crop from the resized video.
Note: The cropping of the same video is not static; it is changing over epochs! 

For validation/inference, 
- Scale the shorter side of size to 224 pixels, and randomly take 3 crops of size 224 x 224 to cover a larger spatial extent within the clip. The final precision is obtained by averaging the softmax scores of these 3 predictions.
